disperser/dataapi/v2/blobs.go

```diff
@@ -447,6 +457,15 @@ func (s *ServerV2) sendBlobFeedResponse(
 		}
 		blobInfo[i].BlobKey = bk.Hex()
 		blobInfo[i].BlobMetadata = createBlobMetadata(blobs[i])
+
+		// Get batch header hash from attestation info if available
+		attestationInfo, err := s.blobMetadataStore.GetBlobAttestationInfo(c.Request.Context(), bk)
```

Comment:
This is inefficient, as it's in blob feed and it'll make hundreds of calls to dynamoDB

Reply:
Profiled this change and it is ~36x slower vs without the batch header hash.